contributing/contributing.html

```diff
@@ -0,0 +1,51 @@
+<ol>
+<li>Read over the <a href="https://github.com/spacetelescope/style-guides/blob/master/guides/jupyter-notebooks.md">style guide</a>. The style parts are guidelines and it is not always necessary to follow them to the letter, but contributions are expected to generally follow this guide.</li>
+<li>Fork your target repository. See the preferred method of doing this on the <a href="https://github.com/spacetelescope/style-guides/blob/master/guides/git-workflow.md">workflow page</a>.</li>
+<li>Create a feature branch to add or modify content (<code>git checkout -b my-new-feature</code>).  You should aim to do one branch (and therefore one Pull Request) per notebook, as that will make review faster and easier. </li>
+<li>Create your notebook, starting with the <a href="https://github.com/spacetelescope/notebooks/tree/master/notebooks/MAST/templates/jupyter_template/jupyter_template.ipynb">the Jupyter Notebook template</a>. This should be organized into the <code>notebooks</code> directory or appropriate sub-directory, with each notebook in its <em>own</em> directory with no other notebooks.</li>
+<li>Develop your notebook by adding and committing it (<code>git add path/to/my/notebook/notebook.ipynb</code>,  <code>git commit -m 'Added some feature'</code>), as many times as necessary to capture your development history.  Note that you must <em>never</em> commit a notebook with executed cells or commit large data files (see the guide).  Always "Clear Ouputs" on cells before adding/committing.</li>
+<li>Add a <code>requirements.txt</code> file next to your notebook.  Each line must be a separate package following <code>pip</code> <a href="https://pip.pypa.io/en/stable/reference/pip_install/#requirement-specifiers">requirements file conventions</a>, listing all the required packages for your notebook, including a "known good version" (e.g., if you wrote the notebook on numpy v1.14.0, the line should be <code>numpy&gt;=1.14.0</code>).<ul>
+<li><strong>A note on supplemental files:</strong> <ul>
+<li>Including supplemental files with your notebook is discouraged, and should only occur if the files are not accessible online (e.g. via MAST).</li>
+<li>See the <a href="https://github.com/spacetelescope/style-guides/blob/master/guides/where-to-put-your-data.md">style guide</a> for more detailed guidance on how to handle supplemental data.</li>
+</ul>
+</li>
+</ul>
+</li>
+<li>Push to your github fork's branch (<code>git push origin my-new-feature</code>).</li>
+<li>Create a new Pull Request from your fork into <code>spacetelescope</code> using the GitHub web site.  Be sure to include a description that's sufficient for someone <em>not in your team</em> to understand the context of the notebook (not all reviewers will be from your team.)</li>
+</ol>
+<p>Once you've created a Pull Request, the CI tests will run on your changes. These ensure that the notebook actually runs.</p>
```

Comment:
The CI part is a little more complicated. Maybe this should link to the page about CI and/or to the documentation Mike is writing?

Reply:
Perhaps the solution is the same as the above; remove the information here and just link to the "style guides" repo. Although, I wonder if it makes sense to have two separate repositories...